api/core/tools/provider/builtin/gitlab/gitlab.yaml

```diff
@@ -5,8 +5,8 @@ identity:
     en_US: Gitlab
```

Comment:
Please replace all `Gitlab` with `GitLab`

Reply:
All of Gitlab has replaced to GitLab